scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -144,20 +149,28 @@ abstract class ScanResultsStorage {
                 maxPoolSize = LocalScanner.NUM_STORAGE_THREADS + 3
             )
 
+            log.info {
+                "Using Postgres storage with URL '${config.url}' and schema '${config.schema}'."
+            }
+
             return PostgresStorage(dataSource)
         }
 
         /**
          * Create a [ClearlyDefinedStorage] based on the [config] passed in.
          */
         private fun createClearlyDefinedStorage(config: ClearlyDefinedStorageConfiguration): ScanResultsStorage =
-            ClearlyDefinedStorage(config)
+            ClearlyDefinedStorage(config).also {
+                log.info { "Using ClearlyDefined storage with URL '${config.serverUrl}'." }
+            }
```

Comment:
I think I would prefer to format this like for less nesting. @sschuberth  would you agree?
```
private fun createClearlyDefinedStorage(config: ClearlyDefinedStorageConfiguration): ScanResultsStorage {
    log.info { "Using ClearlyDefined storage with URL '${config.serverUrl}'." }
    return ClearlyDefinedStorage(config)
}
```

Reply:
Personally, I have a slight preference for the current syntax as it IMO highlights better that creating the storage is the primary thing, and *also* logging is just secondary. But your choice.

Reply:
Ok, I'll keep it.